django/db/models/query_utils.py

```diff
@@ -41,7 +41,9 @@ def __init__(self, *args, _connector=None, _negated=False, **kwargs):
         super().__init__(children=[*args, *sorted(kwargs.items())], connector=_connector, negated=_negated)
 
     def _combine(self, other, conn):
-        if not isinstance(other, Q):
+        from django.db.models.expressions import Subquery
+
+        if not isinstance(other, Q) and not isinstance(other, Subquery):
```

Comment:
I don't think we want to special case `Subquery`, feels like we'll want to duck type on `conditional`

```suggestion
        if not isinstance(other, Q) and not getattr(other, 'conditional', False) is True:
```

Reply:
Thanks @charettes for your suggestion as always:heart: